cardano-cli/CHANGELOG.md

```diff
@@ -1,5 +1,72 @@
 # Changelog for cardano-cli
 
+## 8.23.0.0
+
+- Introduces the `governance action create-hardfork` cmd.
+  (feature, test)
+  [PR 746](https://github.com/IntersectMBO/cardano-cli/pull/746)
+
+- Export friendly function that returns ByteString + export buildShelleyAddress
+  (compatible)
+  [PR 748](https://github.com/IntersectMBO/cardano-cli/pull/748)
+
+- Export toTxOutInAnyEra
+  (compatible)
+  [PR 745](https://github.com/IntersectMBO/cardano-cli/pull/745)
+
+- Enable manual specification of execution units for plutus voting and proposing scripts
+  (feature, breaking)
+  [PR 744](https://github.com/IntersectMBO/cardano-cli/pull/744)
+
+- query stake-distribution: add --output-{json,text} flags
+  (feature, breaking)
+  [PR 743](https://github.com/IntersectMBO/cardano-cli/pull/743)
+
+- Introduce new `cardano-cli latest transaction build-estimate` command which will
+  produce a balanced transaction body without requiring a connection to a live node
+  (feature)
+  [PR 728](https://github.com/IntersectMBO/cardano-cli/pull/728)
+
+- Add `minFeeRefScriptCostPerByte`  to create-protocol-parameter-update governance action command.
+  (feature)
+  [PR 736](https://github.com/IntersectMBO/cardano-cli/pull/736)
+
+- Fixed misunderstanding in generated README file.
```

Comment:
Can you regenerate this file? I've update this PR description.

Reply:
Done :+1: